### PR TITLE
modifying test names to be unique in rgw pacific suites

### PR DESCRIPTION
**Pull request by hmaheswa**
This PR modifies test names to be unique throughout the run in rgw pacific test suites.

Signed-off-by: Hemanth Maheswarla <hmaheswa@hmaheswa.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/pacific/rgw/tier-1-extn_rgw.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ tests:
         config-file-name: test_sts_using_boto.yaml
         timeout: 500
   - test:
-      name: STS Tests
+      name: STS test wit invalid arn in the role's policy
```
Comment (udaysk23) [inline suites/pacific/rgw/tier-1-extn_rgw.yaml:121]: Can you fix the typo here

**Review by udaysk23 — CHANGES_REQUESTED** [review]
Could you please change names in below files
suites/pacific/rgw/tier-1_rgw_ecpool_ms-bucket-object-gc-policy-on-primary.yaml

File: suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml [hunk 2]
```diff
@@ -320,7 +320,7 @@ tests:
             timeout: 300
       desc: test the duration for ordered listing of versioned buckets
       module: sanity_rgw_multisite.py
-      name: Bucket Listing tests
+      name: versoned buckets ordered listing
```
Comment (viduship) [inline suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml:323]: Please fix the tpyo here as well

File: suites/pacific/rgw/tier-1-extn_rgw.yaml (hunk 1 above)
Comment (hmaheswa) [reply suites/pacific/rgw/tier-1-extn_rgw.yaml:121]: okay

File: suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml (hunk 2 above)
Comment (hmaheswa) [reply suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml:323]: okay

File: suites/pacific/rgw/tier-2_rgw_ssl_ecpool-ms-bucket-listing-versioning-on-secondary.yaml [hunk 3]
```diff
@@ -306,7 +306,7 @@ tests:
             timeout: 300
 
   - test:
-      name: Bucket listing test
+      name: Bucket listing flat ordered versionsing on secondary 2
       polarion-id: CEPH-83573545
       desc: test_bucket_listing_flat_ordered_versionsing on secondary
```
Comment (mkasturi18) [inline suites/pacific/rgw/tier-2_rgw_ssl_ecpool-ms-bucket-listing-versioning-on-secondary.yaml:311]: @hmaheswa , could you add this test_bucket_listing_pseudo_ordered.yaml here and change the test name accordingly

Comment (hmaheswa) [reply suites/pacific/rgw/tier-2_rgw_ssl_ecpool-ms-bucket-listing-versioning-on-secondary.yaml:311]: okay madhavi

**Review by udaysk23 — COMMENTED** [review]
@hmaheswa Could you please look into following files and change names accordingly
suites/pacific/rgw/tier-1_rgw_ecpool_ms-bucket-listing-versioning-on-primary.yaml
suites/pacific/rgw/tier-1_rgw_ecpool_ms-bucket-object-gc-policy-on-primary.yaml
suites/pacific/rgw/tier-1_rgw_ms-bucket-listing-versioning-on-secondary.yaml
suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml
suites/pacific/rgw/tier-1_rgw_ssl_multisite_test-upgrade-5-to-latest.yaml
suites/pacific/rgw/tier-1_rgw_ssl_test-upgrade-5-to-latest.yaml
suites/pacific/rgw/tier-1_rgw_test-upgrade-4-to-latest.yaml
suites/pacific/rgw/tier-2_rgw_ssl_ecpool-ms-bucket-listing-versioning-on-secondary.yaml

File: suites/pacific/rgw/tier-1-extn_rgw.yaml [hunk 4]
```diff
@@ -68,7 +68,7 @@ tests:
   # STS tests
 
   - test:
-      name: STS Tests
+      name: STS Tests to Perform assume role on priciple user and perform IOs
```
Comment (ckulal) [inline suites/pacific/rgw/tier-1-extn_rgw.yaml:71]: Can you please correct the typo error in "priciple"

Comment (hmaheswa) [reply suites/pacific/rgw/tier-1-extn_rgw.yaml:71]: Okay Chaithra

**Review by udaysk23 — APPROVED** [review]
LGTM

**Review by ckulal — APPROVED** [review]
lgtm

**Review by anrao19 — APPROVED** [review]
LGTM

**Review by anrao19 — APPROVED** [review]

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**